app/src/androidTest/java/sdp/moneyrun/MapInstrumentedTest.java

```diff
@@ -0,0 +1,72 @@
+package sdp.moneyrun;
+
+import androidx.test.core.app.ActivityScenario;
+
+import com.mapbox.mapboxsdk.geometry.LatLng;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class MapInstrumentedTest {
+
+        @Test
+        public void AddMarkerAddsMarker() {
+            try (ActivityScenario<MapActivity> scenario = ActivityScenario.launch(MapActivity.class)) {
+                float lat = 12f;
+                float lon = 12f;
+                try {
+                    Thread.sleep(2500);
```

Comment:
Why do we need to wait here?

Reply:
Because the map takes time to show and does asynchronous stuff.